HousingSearchApi/V1/Infrastructure/ElasticElasticSearchWrapper.cs

```diff
@@ -17,16 +18,19 @@ public class ElasticSearchWrapper : IElasticSearchWrapper
         private readonly IQueryFactory _queryFactory;
         private readonly IPagingHelper _pagingHelper;
         private readonly ISortFactory _iSortFactory;
+        private readonly IFilterFactory _iFilterFactory;
```

Comment:
would rename to `_filterFactory` (and the same with the one above if possible) for consistency

Reply:
Oh, sure. This is typo I guess. Already fixed